### ISSUE TITLE
upgrade: run-in-terminal.sh doesn't find gnome-console version 45 kgx

### ISSUE THREAD

**Issue by Hi-Phile** [pia-foss/desktop#61]
Problem:
The installer script, `run-in-terminal.sh` does not work with the new gnome-console version 45 which is called `kgx` so after downloading the client update, the install update fails to launch.

![image](https://github.com/pia-foss/desktop/assets/35350630/0e32b3b4-b1da-42b3-b29a-abc8232604b9)

environment variable which shows the terminal:
```
TERM=xterm-256color
TERM_PROGRAM=kgx
TERM_PROGRAM_VERSION=45.0
```
Debug log from PIA client:
```
[2023-10-16 19:59:55.708][631a][client.nativetrayqt][src/linux/nativetrayqt.cpp:482][debug] Menu was triggered "update"
[2023-10-16 19:59:55.708][631a][client.nativehelpers][src/nativehelpers.cpp:417][debug] Running command in terminal "/opt/piavpn/var/update/pia-linux-3.5.1-07760.run"
[2023-10-16 19:59:55.739][631a][common.process][src/exec.cpp:100][warning] (3)"/opt/piavpn/bin/run-in-terminal.sh" "/opt/piavpn/var/update/pia-linux-3.5.1-07760.run"
[2023-10-16 19:59:55.739][631a][client.nativehelpers][src/nativehelpers.cpp:426][debug] Process finished with exit code:  3
[2023-10-16 19:59:55.740][631a][qml][qrc:/components/common/UpdateNotificationStatus.qml:94][error] Failed to execute installer /opt/piavpn/var/update/pia-linux-3.5.1-07760.run
```
Workaround:
You can manually run the downloaded update via `sh /opt/piavpn/var/update/pia-linux-3.5.1-07760.run`

**Comment by kp-john-mair** [comment]
Ironically, I included this in 3.5.1 so it should just work from now on. Thanks for the report!

**Comment by kp-john-mair** [comment]
https://github.com/pia-foss/desktop/blob/master/extras/installer/linux/installfiles/run-in-terminal.sh#L33
